### PR TITLE
Move flow folder, fix tool warning and fix progress bar

### PR DESCRIPTION
**Pull request by 16oeahr**
# Description

Move flow folder, fix tool warning and fix progress bar.

Doc CI check: https://github.com/microsoft/promptflow/actions/runs/8464028391

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/gen_test_data/gen_test_data/common.py [hunk 1]
```diff
@@ -89,7 +90,10 @@ def print_progress(log_file_path: str, process):
 
     logger = get_logger("data.gen")
     logger.info(f"Click '{log_file_path}' to see detailed batch run log. Showing the progress here...")
-    log_pattern = re.compile(r".*execution.bulk\s+INFO\s+Finished (\d+) / (\d+) lines\.")
+    finished_log_pattern = re.compile(r".*execution.bulk\s+INFO\s+Finished (\d+) / (\d+) lines\.")
+    progress_log_pattern = re.compile(
+        r".*execution\.bulk\s+INFO\s+\[Lines]\s+\[Finished:\s+(\d+)]\s+\[Processing:\s+(\d+)]\s+\[Pending:\s+(\d+)]"
```
Comment (chjinche) [inline examples/gen_test_data/gen_test_data/common.py:95]: Can we ignore `[Lines]` from regrex pattern? Maybe more robust.

File: examples/gen_test_data/gen_test_data/common.py [hunk 2]
```diff
@@ -114,20 +118,28 @@ def print_progress(log_file_path: str, process):
...
-                    finished, total = map(int, match.groups())
-                    if progress_bar is None:
-                        progress_bar = tqdm(total=total, desc="Processing", file=sys.stdout)
-                    progress_bar.update(finished - progress_bar.n)
+                    if progress_match:
+                        finished, processing, pending = map(int, progress_match.groups())
+                        total = finished + processing + pending
+                        if progress_bar is None:
+                            progress_bar = tqdm(total=total, desc="Processing", miniters=1, file=sys.stdout)
+                            time.sleep(0.1)  # wait for 0.1 second to show the progress bar on next line
```
Comment (chjinche) [inline examples/gen_test_data/gen_test_data/common.py:131]: due to tqdm issue, need to wait for 0.1 second between progress bar initialization and first update.

File: examples/gen_test_data/gen_test_data/common.py (hunk 1 above)
Comment (16oeahr) [reply examples/gen_test_data/gen_test_data/common.py]: updated. please check again

File: examples/gen_test_data/gen_test_data/common.py (hunk 2 above)
Comment (16oeahr) [reply examples/gen_test_data/gen_test_data/common.py]: updated to use mininterval

**Review by chjinche — APPROVED** [review]

**Pull request merged**